### PR TITLE
feat(release-please): release-please as a new option for release automation (WIP)

### LINKED ISSUE #77: Add release-please an an option for release workflow
https://github.com/googleapis/release-please

### PR DESCRIPTION
**Pull request by Naman-Priyadarshi**
## Pull Request description
Created a new option `release-please` with options `semantic-release', 'release-please', and 'None'. The first one being the default choice. Currently a work in progress

Fixes #77 

## How to test these changes

- `...`

## Pull Request checklists

This PR is a:

- [ ] bug-fix
- [x] new feature
- [ ] maintenance

About this PR:

- [ ] it includes tests.
- [ ] the tests are executed on CI.
- [ ] the tests generate log file(s) (path).
- [x] pre-commit hooks were executed locally.
- [ ] this PR requires a project documentation update.

Author's checklist:

- [ ] I have reviewed the changes and it contains no misspelling.
- [ ] The code is well commented, especially in the parts that contain more
      complexity.
- [ ] New and old tests passed locally.

## Reviewer's checklist

Copy and paste this template for your review's note:

```
## Reviewer's Checklist

- [ ] I managed to reproduce the problem locally from the `main` branch
- [ ] I managed to test the new changes locally
- [ ] I confirm that the issues mentioned were fixed/resolved
```

### CODE REVIEW

**Comment by Naman-Priyadarshi** [comment]
Hey @xmnlab, could you check the changes in the OSL and base profile? I am currently working on refactoring the existing `semantic-release` code so that all options can be used.

File: src/scicookie/profiles/base.yaml [hunk 1]
```diff
@@ -262,6 +262,18 @@ roadmap_document:
     - pytorch-ignite-roadmap
   visible: true
 
+release_workflow:
+  message: Select one option for release automation tool for this project
+  help: "For more information, check:\n  https://osl-incubator.github.io/scicookie/guide/#release-workflow"
+  type: single-choice
+  default: semantic-release
```
Comment (xmnlab) [inline src/scicookie/profiles/base.yaml:269]:
```suggestion
  default: None
```

**Comment by xmnlab** [comment]
@Naman-Priyadarshi , I just added a comment in the base.yaml ... the osl.yaml looks good to me

**Comment by xmnlab** [comment]
Hi @Naman-Priyadarshi , thanks for working on that.
I am closing this PR, but feel free to reopen that if you have more updates.
all the best,

**Pull request closed without merging**